test/tests/transfers/multiapp_copy_transfer/array_variable_transfer/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  issues = '#14391'
+  design = 'MultiAppCopyTransfer.md'
+
+  [./test]
+    type = Exodiff
+    input = master.i
+    exodiff = 'master_out.e master_out_sub0.e'
+
+    requirement = "The system shall support direct copy of multiple nonlinear variables +from+ a sub-application +to+ the main application."
```

Comment:
multiple nonlinear variables? You mean array variable right? I would expect that the former would allow me to do something like `variable = 'u v'` in my input file for the copy transfer.

Reply:
Looks like a copy paste issue to me.

Reply:
Correct, this is only for Array Variables, easy fix.